### DISCUSSION TITLE
Dashboard Setup Help

### DISCUSSION THREAD

**Discussion by agillesp** [datalust/seq-tickets#1727] [Help]
I'd like to see the count of a certain event over different timespans. Let's say the query is `@myevent='target'` How could I setup two dashboard items that show 1. the total count of `target` event in the past 24 hours and 2. the total count of `target` event in the past year? The dropdown filters in the upper-right (`Last 24 hours by 5 minutes`, for ex) apply globally; which means my two counters are going to show the same result. Can the sample span setting be applied to individual dashboard items, or even to only one dashboard and not another? I feel like I'm missing something here. Thanks!

**Comment by nblumhardt** [comment]
Hi! Thanks for reaching out. This is a limitation in the current dashboarding system, there's no way to achieve this currently, though we're keen to add more flexibility here in future.

**Reply by agillesp** [reply]
OK, thanks. Would definitely be a nice feature to be able to set the time span, grouping span, and refresh rate per each widget. For now I've found that date filtering in the `where` clause gets me mostly to what I'm looking for. For example, events for "just today", I've used this:

`@Timestamp > DatePart(Now(), 'date', OffsetIn('America/Denver', Now()))`
